src/temp/Project/Filter.tsx

```diff
@@ -0,0 +1,48 @@
+import FilterIcon from '../icons/FilterIcon';
+import { Button } from '@mui/material';
+import style from './Filter.module.scss';
+import { useTranslation } from 'next-i18next';
+
+interface FilterProps {
+  activeFilter: boolean;
+  projectList: string[];
+}
+
+export const FilterDropDown = ({ activeFilter, projectList }: FilterProps) => {
+  const { t } = useTranslation(['projectDetails']);
+  return (
+    <>
+      {activeFilter && projectList.length > 0 ? (
+        <div className={style.projectListMainContainer}>
+          <div className={style.container}>
+            <h1>{t('projectDetails:allProjects')}</h1>
+
+            {projectList.map((projectType) => {
+              return (
+                <>
+                  <hr />
+                  <div className={style.projectName}>{projectType}</div>
+                </>
+              );
+            })}
+          </div>
+        </div>
+      ) : (
+        <></>
+      )}
+    </>
+  );
+};
+
+const Filter = ({ activeFilter, projectList }: FilterProps) => {
+  return (
```

Comment:
1. It might be better to keep `activeFilter` in state instead of props. Consider renaming as well to `isFilterOpen` as well, as `activeFilter` suggests that a filter has been applied
2. How do we know if a filter is applied or all projects are selected? This information (selected filter) is probably relevant in the parent component as well in order to apply the filter to the list.

Reply:
When a filter is applied, only those projects that fulfill the filter criteria will be visible in the project list.
![image](https://github.com/Plant-for-the-Planet-org/planet-webapp/assets/101264823/66666896-bf03-4f42-905e-8441c4f254ad)